[DC 2015] Sponsor updates

markcornick opened the pull request.

phrawzty commented:
Did you mean to move Red Hat as well? Just want to verify since the commit message only mentions Delphix.

markcornick commented:
Yes, the Red Hat move was intentional - I added a second commit to this PR, rebased into one, but wasn't able to edit the PR title on the latent connection I was using yesterday. Thanks!
